.env.example

```diff
@@ -11,3 +11,4 @@ KEYCARD_SECRET=
 # Sentry (Optional)
 SENTRY_DSN=
 SENTRY_TRACE_SAMPLE_RATE=
+BROVIDER_URL=https://rpc.snapshot.org
```

Comment:
Maybe better to have it as a const variable?

Easy to find/maintain it from code than from env. as it is not often changed

Reply:
Or use both method, keep in .env so we can still override it via .env var (such as when using with Docker), and redefine locally on top of file, to set a default value 

`const broviderUrl = process.env.BROVIDER_URL  || 'https://rpc.snapshot.org'`

Leaving the .env var optional